From 17e4f04b361ed90d64aec03ecdf97116fe1cb6bf Mon Sep 17 00:00:00 2001
From: hugo.delacruz@live.com.mx <hugo.delacruz@live.com.mx>
Date: Wed, 22 Sep 2010 10:20:49 +0200
Subject: [PATCH] Se agrego titulo

---
 README |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/README b/README
index e69de29..100b938 100644
--- a/README
+++ b/README
@@ -0,0 +1 @@
+README
\ No newline at end of file
-- 
1.7.0.2.msysgit.0

